src/wp-content/themes/twentyseventeen/functions.php

```diff
@@ -439,8 +439,8 @@ function twentyseventeen_colors_css_wrap() {
 		$customize_preview_data_hue = 'data-hue="' . $hue . '"';
 	}
 	?>
-	<style type="text/css" id="custom-theme-colors" <?php echo $customize_preview_data_hue; ?>>
-		<?php echo twentyseventeen_custom_colors_css(); ?>
+	<style type="text/css" id="custom-theme-colors" <?php echo esc_attr( $customize_preview_data_hue ); ?>>
+		<?php echo wp_strip_all_tags( twentyseventeen_custom_colors_css() );  ?>
```

Comment:
```suggestion
		<?php echo wp_strip_all_tags( twentyseventeen_custom_colors_css() ); ?>
```

The PHP coding standards GitHub action caught that there are two spaces after the semicolon when it should only be one.